packages/manager/src/features/linodes/LinodesCreate/LinodeCreateContainer.tsx

```diff
@@ -404,6 +410,12 @@ class LinodeCreateContainer extends React.PureComponent<CombinedProps, State> {
     }));
   };
 
+  handleShowApiAwarenessModal = () => {
```

Comment:
Consider updating the function name to better match the behavior

```suggestion
  handleToggleApiAwarenessModal = () => {
```